analysewrite_test.go

```diff
@@ -140,7 +141,7 @@ func TestUnderscore(t *testing.T) {
 }
 
 func TestNonAscii(t *testing.T) {
-	anal(t, `digraph {	label=T�th }`)
+	anal(t, `digraph {	label=Tóth }`)
```

Comment:
> Did this cause a problem?

It did indeed. The Gocc generated lexer will return an error stating that the rune is a [Unicode replacement](https://golang.org/pkg/unicode/#ReplacementChar) rune, which signals an error in decoding. Replacing the character with an actual Unicode character resolved the issue.